detection_rules/etc/_config.yaml

```diff
@@ -18,6 +21,12 @@ files:
 #         ├── packages.yml
 #         ├── stack-schema-map.yaml
 #         └── version.lock.json
+#     └── actions
+##         ├── action_1.toml
+##         ├── action_2.toml
+#     └── exceptions
+##         ├── exception_1.toml
+##         ├── exception_2.toml
```

Comment:
Does this mean users will have to manually list every actions/exceptions list file in the config ?